### PR TITLE
tests: Add docker run with stdin.

### LINKED ISSUE #853: add stdin test for docker run
add a docker running integration test with stdin

### PR DESCRIPTION
**Pull request by GabyCT**
Test stdin while running docker run.

Fixes #853

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -92,3 +92,44 @@ func (c *Command) Run() (string, string, int) {
 		return stdout.String(), stderr.String(), exitCode
```
Comment (devimc) [inline command.go]: could you please modify ```Run() ``` to use the new function ```RunWithPipe() ```

```go
func (c *Command) Run() (string, string, int) {
    return c.RunWithPipe(nil)
}
```

File: docker.go [hunk 2]
```diff
@@ -51,10 +52,24 @@ func runDockerCommandWithTimeout(timeout time.Duration, command string, args ...
 	return cmd.Run()
```
Comment (devimc) [inline docker.go]: also please update ```runDockerCommandWithTimeout```

```
func runDockerCommandWithTimeout(timeout time.Duration, command string, args ...string) (string, string, int) {
   return runDockerCommandWithTimeoutAndPipe(nil, timeout, command, args...)
}
```

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**